examples/ondeck/mysql/models.go

```diff
@@ -11,46 +11,46 @@ import (
 	"time"
 )
 
-type VenuesStatus string
+type VenueStatus string
```

Comment:
This name changed because the table is called `venue` not `venues`. The changed name wasn't taken into account before.